backend/app/adapter/db/url.go

```diff
@@ -78,6 +79,47 @@ WHERE "%s"=$1;`,
 	return url, nil
 }
 
+// GetURLByVisibility implements retrieval of URL associated with userEmail based on type specified
+// TODO: Add urlType based querying after column addition
+func (u URLSql) GetURLByVisibility(userEmail, urlType string) ([]search.URLSearchResponse, error) {
+	statement := fmt.Sprintf(`
+	SELECT *
```

Comment:
```suggestion
	SELECT *
```

Let's replace * with explicit column name to prevent from breaking this function when adding extra columns to the table.

Reply:
Using `original_url` as coulmn_name now.